manifests/config.pp

```diff
@@ -1,63 +1,44 @@
 # private
-class collectd::config (
-  $collectd_hostname                    = $collectd::collectd_hostname,
-  Stdlib::Absolutepath $config_file     = $collectd::config_file,
-  $conf_content                         = $collectd::conf_content,
-  Boolean $fqdnlookup                   = $collectd::fqdnlookup,
-  Boolean $has_wordexp                  = $collectd::has_wordexp,
-  Array $include                        = $collectd::include,
-  Boolean $internal_stats               = $collectd::internal_stats,
-  Integer $interval                     = $collectd::interval,
-  Stdlib::Absolutepath $plugin_conf_dir = $collectd::plugin_conf_dir,
-  $plugin_conf_dir_mode                 = $collectd::plugin_conf_dir_mode,
-  $recurse                              = $collectd::recurse,
-  $config_group                         = $collectd::config_group,
-  String $config_owner                  = $collectd::config_owner,
-  $purge                                = $collectd::purge,
-  Boolean $purge_config                 = $collectd::purge_config,
-  Integer $read_threads                 = $collectd::read_threads,
-  Integer $timeout                      = $collectd::timeout,
-  Array $typesdb                        = $collectd::typesdb,
-  $write_queue_limit_high               = $collectd::write_queue_limit_high,
-  $write_queue_limit_low                = $collectd::write_queue_limit_low,
-  Integer $write_threads                = $collectd::write_threads,
-) {
+class collectd::config inherits collectd {
```

Comment:
`inherits` will be removed when the ERB template is replaced with a EPP in a future PR